(feat) Updates for RN 6.x (#168)
embrace-io/embrace-docs: docs/react-native/integration/integration-steps.md

# Integration Steps

> 1. [**Login to the Embrace Dashboard.**](/react-native/integration/login-embrace-dashboard) Your data is available in the dashboard. It is important that you have this open while working on the integration to monitor progress. If you are looking to run Embrace and sending data outside of our Backend you may want to jump into how to [Send data outside Embrace](/react-native/integration/send-data-custom-backend/) section.

Reviewer: ```suggestion
1. [**Login to the Embrace Dashboard.**](/react-native/integration/login-embrace-dashboard) Your data is available in the dashboard. It is important that you have this open while working on the integration to monitor progress. If you are looking to use the SDK without sending data to Embrace you may want to jump into the how to [Send data outside Embrace](/react-native/integration/send-data-custom-backend/) section.
```
